ProjectName/src/screens/PaymentDetails/PaymentDetailsForm/payment-details-form.styles.ts

```diff
@@ -0,0 +1,40 @@
+import {StyleSheet} from 'react-native';
```

Comment:
Please, don't forget putting empty line between `imports` and other part of the file.